src/main/java/datawave/query/util/MetadataHelper.java

```diff
@@ -95,6 +85,8 @@
     
     protected static final Text PV = new Text("pv");
     
+    public static final String COL_QUAL_PREFIX = "compressed-";
```

Comment:
We are going against convention here by using a '-' instead of a null byte.  Also using word "compressed" seems like a waist of bytes.  Perhaps something shorter?